pkg/util/patroni/patroni.go

```diff
@@ -102,3 +103,14 @@ func (p *Patroni) SetPostgresParameters(server *v1.Pod, parameters map[string]st
 	}
 	return p.httpPostOrPatch(http.MethodPatch, apiURL(server)+configPath, buf)
 }
+
+//SetPostgresParameters sets Postgres options via Patroni patch API call.
+func (p *Patroni) EditStandby(server *v1.Pod, options map[string]string) error {
+	buf := &bytes.Buffer{}
+	err := json.NewEncoder(buf).Encode(map[string]map[string]interface{}{"standby_cluster": {"restore_command": options}})
```

Comment:
shouldn't it be `{"standby_cluster": options}` ?

Reply:
ping, is it a WIP PR?

Reply:
Well, yes. I am sorry for the inconvenience it might have caused because of multiple commits. Will update the status as need review as soon as it is moves to the next step.